examples/apis/customer/create/main.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mercadopago/sdk-go/pkg/config"
+	"github.com/mercadopago/sdk-go/pkg/customer"
+)
+
+func main() {
+	accessToken := "{{ACCESS_TOKEN}}"
+
+	cfg, err := config.New(accessToken)
+	if err != nil {
+		fmt.Println(err)
+		return
+	}
+
+	email := "{{EMAIL}}"
+	req := customer.Request{Email: email}
+
+	client := customer.NewClient(cfg)
+	res, err := client.Create(context.Background(), req)
```

Comment:
res é um nome muito genérico, como conversamos acredito que algo assim fica melhor:
```suggestion
	cust, err := client.Create(context.Background(), req)
```
@eltinMeli @gdeandradero